src/components/Root/index.tsx

```diff
@@ -31,6 +33,9 @@ const RootConsumer = (): React.ReactElement | null => {
   const [isError, setIsError] = useState<boolean>(false)
 
   useEffect(() => {
+    // Initialize the SDK
+    setBaseUrl(GATEWAY_URL)
```

Comment:
Updating to the new SDK required the `baseUrl` to be set. Alongisde this, passing of `GATEWAY_URL` have been removed app-wide.